### DISCUSSION TITLE
npm: command not found -- catalogue ansible

### DISCUSSION THREAD

**Discussion by prabalark** [learndevopsonline/learndevopsonline#1928] [Q&A]
hi team , 
i am getting error while doing catalogue install in ansible 
catalogue : 54.242.96.107
terraform : 3.84.21.225
jenkins : 54.159.107.44

![image](https://github.com/user-attachments/assets/5886ce7f-261b-4ef0-9b2c-220f693d19b9)

**Comment by prabalark** [comment]
![image](https://github.com/user-attachments/assets/10c2a5c5-c316-46e9-bb2c-5a6b4895f3d9)

**Comment by chaithanya1812** [comment]
Hi @prabalark . Document got updated please have a look of installing nodejs.
https://learndevopsonline.github.io/learndevopsonline/build/docs/RoboShop-Project/catalogue
![image](https://github.com/user-attachments/assets/fc317fad-1d88-4d65-ab46-77f193897bd4)

**Reply by prabalark** [reply]
hi chaithanya , how to write syntax for both disable & enable in ansible

**Reply by prabalark** [reply]
i find this syntax is this correct
![image](https://github.com/user-attachments/assets/075c8671-6ec2-4fb4-a6b6-9461e8c885f5)

**Comment by chaithanya1812** [comment]
Hi @prabalark, 
Use shell module

**Reply by prabalark** [reply]
ansible.builtin.shell : dnf module disable nodejs
ansible.builtin.shell : dnf module enable nodejs>18

is this correct

**Reply by prabalark** [reply]
![image](https://github.com/user-attachments/assets/d851576a-048e-4faf-900f-3b2c1e4e4145)

**Reply by prabalark** [reply]
while run ansible getting below error 
![image](https://github.com/user-attachments/assets/827cd5ad-2bb8-4636-b7ec-adcae04c89a4)

**Comment by prabalark** [comment]
it is working  , in shell module ,-y also has to keep 
 ![image](https://github.com/user-attachments/assets/3a10f6e4-b829-4098-96c0-6300ac6faed7)
